src/vivarium_cluster_tools/vipin/perf_report.py

```diff
@@ -189,6 +191,86 @@ def print_stat_report(perf_df: pd.DataFrame, scenario_cols: list):
+    """Append performance data to the central logs."""
+    perf_df = perf_df.reset_index()
+    # add location data to perf_df
+    artifact_path_col = "scenario_input_data_artifact_path"
+    if artifact_path_col in perf_df.columns:  # if we parallelized across artifact paths
+        perf_df["location"] = perf_df[artifact_path_col].apply(
+            lambda filepath: Path(filepath).stem
+        )
+    else:  # else get from output directory
+        perf_df["location"] = output_directory.parents[3].stem
+
+    ## aggregate scenario information into one column
+    all_scenario_cols = [col for col in perf_df.columns if col.startswith("scenario_")]
+    # remove duplicate scenario information
+    unique_scenario_cols = [
+        col for col in all_scenario_cols if not col.startswith("scenario_run_configuration")
+    ]
+    perf_df["scenario_parameters"] = perf_df[unique_scenario_cols].to_dict(orient="records")
+    perf_df["scenario_parameters"] = perf_df["scenario_parameters"].apply(json.dumps)
+    perf_df = perf_df.drop(all_scenario_cols, axis=1)
+
+    # append child job data
+    NUM_ROWS_PER_FILE = 100_000
+    performance_logs_dir = Path("/mnt/team/simulation_science/pub/performance_logs/")
+    log_files = glob.glob(Path(performance_logs_dir).as_posix() + "/log_summary_*.csv")
+    sorted_files = sorted(log_files, key=lambda x: Path(x).stem.replace("log_summary_", ""))
+    most_recent_file_path = sorted_files[-1]
+    most_recent_data = pd.read_csv(most_recent_file_path)
+
+    data_fits_in_file = (len(most_recent_data) + len(perf_df)) <= NUM_ROWS_PER_FILE
+
+    if data_fits_in_file:
+        first_file_with_data = most_recent_file_path
+        perf_df.to_csv(most_recent_file_path, mode="a", header=False, index=False)
+    else:
+        # fill up the most recent file (possibly with 0 rows)
+        first_file_with_data = most_recent_file_path
+        rows_to_append = NUM_ROWS_PER_FILE - len(most_recent_data)
+        perf_df[:rows_to_append].to_csv(
+            most_recent_file_path, mode="a", header=False, index=False
+        )
+        perf_df = perf_df[rows_to_append:]
+
+        most_recent_index = int(Path(most_recent_file_path).stem.replace("log_summary_", ""))
+        new_index = most_recent_index + 1
+
+        # record first file with data as new file if no data was appended to most recent file
+        if rows_to_append == 0:
+            formatted_new_index = "{:04}".format(new_index)
+            first_file_with_data = (
+                Path(performance_logs_dir) / f"log_summary_{formatted_new_index}.csv"
+            )
+
+        num_appends = math.ceil(len(perf_df) / NUM_ROWS_PER_FILE)
+
+        for append_num in range(num_appends):
+            formatted_new_index = "{:04}".format(new_index)
+            new_file = Path(performance_logs_dir) / f"log_summary_{formatted_new_index}.csv"
+            start_idx = NUM_ROWS_PER_FILE * append_num
```

Comment:
this doesn't work if you've removed some of the first rows by appending to an existing log file. I think you need to reset the index above at line 236, e.g. `perf_data = perf_data[rows_to_append:].reset_index(drop=True)`